.golangci.yaml

```diff
@@ -8,6 +8,11 @@ issues:
     - "^G304: Potential file inclusion via variable"
     - "^G306: Expect WriteFile permissions to be 0600 or less"
     - "^G301: Expect directory permissions to be 0750 or less"
+  exclude-rules:
+    # Dynamic errors are OK in main packages. Packages should use sentinels.
+    - path: "^([^/]*|build-tools/.*|cmd/.*)$"
```

Comment:
I'm not sure how the `cmd/` part works. It is anchored to the start of the path, yet the file with the lint error is `learn/cmd/levy/ain.go`.

Ah, I see now - for `learn` you first `cd learn`. I would consider duplicating this rule for the things in `learn/` with an appropriate comment.

Reply:
ok, fair enough.
updated. I've also added a Makefile cleanup commit to this pr to used `go CMD -C dir`
everywhere. seems related.